api/http/proxy/factory/docker/volumes.go

```diff
@@ -44,6 +46,11 @@ func (transport *Transport) volumeListOperation(response *http.Response, executo
 	if responseObject["Volumes"] != nil {
 		volumeData := responseObject["Volumes"].([]interface{})
 
+		for _, volumeObject := range volumeData {
+			volume := volumeObject.(map[string]interface{})
+			volume[volumeObjectIdentifier] = volume["Name"].(string) + volume["CreatedAt"].(string)
```

Comment:
You'll need to validate that the value returned by `volume["Name"]` exists before your try to cast it via `.(string)`. If it do not exist for some reason then the program will panic here.

This is why we're using the `if responseObject["Property"] != nil { ... }` pattern in this package.